x-pack/plugins/enterprise_search/common/types/index.ts

```diff
@@ -32,6 +29,11 @@ export interface ConfiguredLimits {
   workplaceSearch: WorkplaceSearchConfiguredLimits;
 }
 
+export interface Access {
```

Comment:
`EnterpriseSearchAccess`? I honestly am not sure how much I feel like we should be namespacing these in some way more but `Access` feels very generic to me

Reply:
Maybe `ProductAccess`? I think the thing I struggle with with the `EnterpriseSearch` prefix is that it feels it either potentially applies to 1. the entire solution, or 2. both products (in this case not correct), or 3. the specific `Enterprise Search` overview plugin.